metadata-ingestion/setup.py

```diff
@@ -226,6 +226,14 @@
     "clickhouse-sqlalchemy>=0.2.0,<0.2.5",
 }
 
+dataplex_common = {
+    "google-cloud-dataplex",
+    # Pinned to 0.2.2 because 0.3.0 changed the import path from
+    # google.cloud.datacatalog.lineage_v1 to google.cloud.datacatalog_lineage,
```

Comment:
we have even 0.4.0 for https://pypi.org/project/google-cloud-datacatalog-lineage/#history

where are those `google.cloud.datacatalog.lineage_v1` imports that prevent us to upgrade? it's us or some other component?

I'm concerned on being pinned to a 2 years old version

Reply:
Deferring for a later PR makes sense

Maybe, just mention in the comment that the one holding this dependency back is `bigquery_v2/lineage.py`.